source/includes/steps-configure-windows-service-for-mongodb.yaml

```diff
@@ -67,7 +67,7 @@ title: Connect to MongoDB.
 stepnum: 7
 ref: connect
 pre: |
-  To connect to MongoDB through the :binary:`~bin.mongo.exe <mongo>` shell,
+  To connect to MongoDB through the :binary:`~bin.mongo.exe` shell,
```

Comment:
this I think renders as exe

You probably want :binary:`mongo.exe <bin.mongo.exe>`

Reply:
The `mongod.exe` did render as `exe`, so I filed a bug with @i80and and he fixed it because `mongod.exe` now shows normally. Why `mongo.exe` does not? I will check.

Reply:
Scratch that. I see the problem. Working on a fix now

Reply:
@atsansone There's no `mongo.exe` definition, since `mongo` doesn't have a Windows-specific page.

You'll have to do ```
:binary:`mongo.exe <bin.mongo>`
```